### DISCUSSION TITLE
Render images

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#3344] [Help]
I have image names stored in the database. When I try to render it on the edge it's not working.  
`   @each(item in data)
      <p>{{item.src}}</p>
      <img src="{{ asset('assets/images/{{item.src}}') }}" class="card-img-bottom card-img-top" alt="image1">
      <div class="card-body">
        <h5 class="card-title text-center">{{ item.title }}</h5>
      </div>
      @else
      <p class="fs-4 fw-bold text-center">No pins found</p>
      @endeach`

Error: Cannot find path for "assets/images/{{item.src}}" asset. Make sure you are compiling assets.
Here, src is image name. How can I render images by the name? Can anyone help me out here?

**Comment by etc-austin** [comment]
This is a guess based on other node projects I've worked on so it may not work but this is what I would try. 
back ticks instead of normal ticks

`
{{ asset(`\`assets/images/\`${item.src}) }}
